ROADMAP.md

```diff
@@ -10,75 +10,84 @@ We use the [milestone](https://github.com/rook/rook/milestones) feature in Githu
 * CockroachDB
-  * Secure deployment using certificates [#1809](https://github.com/rook/rook/issues/1809)
   * Helm chart deployment [#1810](https://github.com/rook/rook/issues/1810)
 * EdgeFS
   * Declare EdgeFS CRDs to be Beta v1 [#2506](https://github.com/rook/rook/issues/2506)
   * Automatic host validation [#2409](https://github.com/rook/rook/issues/2409)
-  * Target VDEVs
-    * Support for flexible write I/O “sync” option [#2367](https://github.com/rook/rook/issues/2367)
+  * Allow for more performance vs. availability tradeoff configuration [#2367](https://github.com/rook/rook/issues/2367)
   * Object
     * OpenStack/SWIFT CRD [#2509](https://github.com/rook/rook/issues/2509)
     * Support for S3 bucket as DNS subdomain [#2510](https://github.com/rook/rook/issues/2510)
-  * Block (iSCSI)
-    * Support Block CSI [#2507](https://github.com/rook/rook/issues/2507)
-  * Mgr
-    * Support for Prometheus Dashboard and REST APIs [#2401](https://github.com/rook/rook/issues/2401)
-    * Support for Management GUI with automated CRD wizards [#2508](https://github.com/rook/rook/issues/2508)
+  * Support Block (iSCSI) CSI [#2507](https://github.com/rook/rook/issues/2507)
+  * Support for Prometheus Dashboard and REST APIs [#2401](https://github.com/rook/rook/issues/2401)
+  * Support for Management GUI with automated CRD wizards [#2508](https://github.com/rook/rook/issues/2508)
+  * Failure domains and zoning provisioning support [#2513](https://github.com/rook/rook/issues/2513)
+  * Multi-Namespace clusters support [#2878](https://github.com/rook/rook/issues/2878)
+  * Embedded mode support [#2810](https://github.com/rook/rook/issues/2810)
 * Minio
   * Helm chart deployment [#1814](https://github.com/rook/rook/issues/1814)
+
+
+## Initial planning for Rook 1.1
+
+* Cassandra
+  * Continue to implement Cassandra design [2294](https://github.com/rook/rook/issues/2294)
+* Ceph
+  * Remove support for Ceph Luminous
+  * Stable release of Ceph-CSI plug-in (feature parity with FlexVolume)
+  * Mon placement respects failure domains [#2603](https://github.com/rook/rook/issues/2603)
+  * User-modifiable configuration at runtime [#2470](https://github.com/rook/rook/pull/2470/files)
+  * Support modified CRUSH maps [#2514](https://github.com/rook/rook/issue
+  * OSDs
+    * Run on arbitrary PVs (local storage) as an alternative to host path [#919](https://github.com/rook/rook/issues/919)
+    * Leverage node labels for CRUSH locations [#1366](https://github.com/rook/rook/issues/1366)
+  * Orchestrate multi-site replication [#1584](https://github.com/rook/rook/issues/1584)
+  * Document a safe shutdown procedure [#2517](https://github.com/rook/rook/issues/2517)
 * NFS
   * Dynamic NFS provisioning [#2062](https://github.com/rook/rook/issues/2062)
-  * Client access control [#2283](https://github.com/rook/rook/issues/2283)
 
-## Beyond 1.0
+## Future improvements
```

Comment:
we could also capture some more of the ideas that we identified as GSoC projects, all those projects have value in the near future.  Full list: https://github.com/cncf/soc#rook

Reply:
How's this look?

Reply:
I actually don't think that Google Summer of Code should be mentioned explicitly here.  Instead, I would integrate these project ideas, along with their issue numbers, into the natural hierarchy of the roadmap, similar to how it looks in the current version.